packages/turf-ellipse/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@turf/ellipse",
+  "version": "5.0.4",
```

Comment:
Could you change the version to `5.0.0` for the first publish, Lerna will decide the patch version.